src/Core/Domain/Configuration/User/Model/User.php

```diff
@@ -34,7 +32,9 @@ class User
                  MIN_EMAIL_LENGTH = 1,
                  MAX_EMAIL_LENGTH = 255,
                  MIN_THEME_LENGTH = 1,
-                 MAX_THEME_LENGTH = 100;
+                 MAX_THEME_LENGTH = 100,
+                 THEME_LIGHT = 'light',
+                 THEME_DARK = 'dark';
```

Comment:
```suggestion
```
This constants seems useless since they are defined in parent class with the same values.

Reply:
I let the constant in both class to be more consistent when we call them, I think it could be odd to call NewUser::THEME_LIGHT for example, while checking the theme on a User and not a NewUser